x-pack/plugins/ingest_manager/common/types/models/epm.ts

```diff
@@ -67,6 +67,15 @@ export interface RegistryPackage {
   path: string;
 }
 
+interface RegistryImage {
+  // https://github.com/elastic/package-registry/blob/master/util/package.go#L74
+  // says src is potentially missing but I couldn't find any examples
+  // it seems like src should be required. How can you have an image with no reference to the content?
+  src: string;
```

Comment:
@ruflin Won't an image always have a `src`? Or, rather, shouldn't `src` be required?

Reply:
Yes: https://github.com/elastic/package-registry/pull/245